CHANGELOG.md

```diff
@@ -12,6 +12,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   rather than `null` if the mime type could not be found.
 - The `@cumulus/ingest/lock.removeLock` function now returns `undefined`
 
+### Added
```

Comment:
Seems like a bad CHANGELOG merge

Reply:
I think it's actually a bad github diff - note this isn't the version update PR, just pulling 2116  into the branch.    That looks like master's changelog https://github.com/nasa/cumulus/blob/master/CHANGELOG.md in that the 2116 content is in 'added'

Reply:
Version up PR (based to this branch) https://github.com/nasa/cumulus/pull/1819